dev-python/polib/polib-1.0.7.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
+
+inherit distutils-r1
+
+DESCRIPTION="A library to manipulate gettext files (.po and .mo files)"
+HOMEPAGE="https://bitbucket.org/izi/polib/wiki/Home"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+RDEPEND=""
+DEPEND="${RDEPEND}
+	doc? ( dev-python/sphinx )
+"
+
+python_compile_all() {
+	if use doc; then
+		cd docs || die
+		emake html
+	fi
+}
+
+python_test() {
+	python tests/tests.py || die
+}
+
+python_install_all() {
+	local DOCS="CHANGELOG README.rst"
```

Comment:
It's documented as array: https://github.com/gentoo/gentoo/blob/master/eclass/distutils-r1.eclass#L124